FRAuth/FRAuth/AppIntegrity/FRAppAttestDomainModal.swift

```diff
@@ -0,0 +1,140 @@
+//
+//  FRAppIntegrityDomainModal.swift
+//  FRAuth
+//
+//  Copyright (c) 2023 ForgeRock. All rights reserved.
+//
+//  This software may be modified and distributed under the terms
+//  of the MIT license. See the LICENSE file for details.
+//
+
+import Foundation
+import CryptoKit
+import DeviceCheck
+
+/// Protocol to override attestation
+@available(iOS 14.0, *)
+public protocol FRAppAttestation {
```

Comment:
Should we consider FRAppAttestation , FRAppAttestDomainModal, FRAppAttestService as internal class?
Do we need developer to have access of those interfaces?

Reply:
@witrisna i think the constructor injection in once of class made all the cascading class to public, i have provided the fix, this will be internal . if you remove public its default to internal in swift . Good Catch